.github/workflows/check_push.yml

```
@@ -0,0 +1,19 @@
+name: Check branch conformity
+on:
+  push:
+
+jobs:
+  prevent-fixup-commits:
+      runs-on: ubuntu-latest
+      env:
+        target: debian-bookworm
+        distro: debian
+        version: bookworm
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v3
+        - name: prevent fixup commits
+          run: |
+            git fetch origin
+            git status
+            git log origin/main..HEAD | grep FIXUP && exit 1 || true
```

```suggestion
            git log origin/main..HEAD | grep -ie '^fixup\|^wip' && exit 1 || true
```

I suggest including both wip commits, and `fixup!` commits that Git produces.

---

Note that an even better approach is `^\s*` (match any whitespace from the start of the line), but we don't need to be pedantic here.